20 Dataview Queries/List non existing, linked pages.md

```diff
@@ -55,6 +55,40 @@ for (let page in unresolvedLinksMap) {
 dv.table(["Unresolved Link", "Contained in"], Object.values(res).map(l => [l.link, l.usages]));
 ```
 
+#### A variant that sorts the results by count of references
```

Comment:
Variants use a three level heading and you're already under the heading "Variant", so the repetition isn't necessary, in my opinion. My proposal is to slightly rephrase:
```suggestion
### Sort the results by count of references
```

Reply:
Looks fine! And two days is not that long right, thanks for coming back to me in the first place ;)

Reply:
Would you be so kind and add + push the suggested change to your branch? Then I can merge back your contribution - thanks!